sandbox/icons.html.example

```diff
@@ -4,7 +4,7 @@
   <meta charset="utf-8" />
   <title>Video.js Icons Sandbox</title>
 
-  <link href="../build/temp/video-js.css" rel="stylesheet" type="text/css">
+  <link href="../build/video-js.css" rel="stylesheet" type="text/css">
```

Comment:
Is this one intentionally using `build` instead of `dist`?

Reply:
nope. Probably got confused about it, heh.